gui/packages/desktop/src/renderer/app.js

```diff
@@ -360,18 +360,78 @@ export default class AppRenderer {
     actions.settings.updateAutoConnect(autoConnect);
   }
 
+  async _getAppComponentsVersions() {
+    const daemonVersion = await this._daemonRpc.getCurrentVersion();
```

Comment:
That's a cheap call so we'll do it twice, once in `fetchCurrentVersion` and then second time in `fetchLatestVersionInfo`. The reason for that is apparently the networking latency, also if there is inconsistency we could report it way before the latest version info is received from the API (if ever)